scipy/_lib/tests/test__util.py

```diff
@@ -343,8 +351,9 @@ def test_contains_nan_with_strings(self):
         data4 = np.array([["1", 2], [3, np.nan]], dtype='object')
         assert _contains_nan(data4)
 
+    @pytest.mark.skip_xp_backends("jax.numpy", reason="lazy backends tested separately")
```

Comment:
TODO add dask

Reply:
I'm wondering if eager `jax.numpy` still has value to test, since it works and the JIT tests are probably going to end up being very slow. Not sure how much slower - what are you seeing for this test?

Reply:
It looks like there is no measurable performance difference.

```
$ python dev.py test -b jax.numpy -t scipy._lib.tests.test_array_api -t scipy._lib.tests.test__util -- -k jax

main (no jit)
15 passed, 2 skipped, 51 deselected in 7.71s
This PR (jit)
13 passed, 5 skipped, 51 deselected in 7.89s 
```

I'm _fairly_ sure that I'm not caching on disk; `jax.config.jax_compilation_cache_dir` returns None.

Reply:
Great, let's give it a try as is then.